### PR TITLE
Filters

### PR DESCRIPTION
**Pull request by kabir**
The full set of PRs is:
https://github.com/wildfly/wildfly-cekit-modules/pull/71
https://github.com/wildfly/temp-eap-modules/pull/48
https://github.com/wildfly/temp-openshift-image/pull/24

### CODE REVIEW

File: jboss/container/wildfly/launch/filters/module.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+schema_version: 1
+name: jboss.container.wildfly.launch.filters
+version: '1.0'
+description: Configures the admin user and associated management realm
```
Comment (jfdenise) [inline jboss/container/wildfly/launch/filters/module.yaml]: Description seems copy/paste

Comment (kabir) [reply jboss/container/wildfly/launch/filters/module.yaml]: You are quick :)

File: jboss/container/wildfly/galleon/fp-content/java/module.yaml [hunk 2]
```diff
@@ -6,3 +6,7 @@ description: java content to add to fp.
 execute:
 - script: configure.sh
   user: 185
+  
+modules:
```
Comment (jfdenise) [inline jboss/container/wildfly/galleon/fp-content/java/module.yaml]: This was already merged in https://github.com/wildfly/wildfly-cekit-modules/pull/70/files

File: jboss/container/wildfly/galleon/fp-content/jolokia/module.yaml [hunk 3]
```diff
@@ -6,3 +6,7 @@ description: jolokia content to add to fp.
 execute:
 - script: configure.sh
   user: 185
+
+modules:
```
Comment (jfdenise) [inline jboss/container/wildfly/galleon/fp-content/jolokia/module.yaml]: https://github.com/wildfly/wildfly-cekit-modules/pull/70/files

**Pull request merged**